lambda-s3/inputFile.txt

```diff
@@ -0,0 +1,38 @@
+{
+  "Records":[
+    {
+      "eventVersion":"2.0",
+      "eventSource":"aws:s3",
+      "awsRegion":"us-west-2",
+      "eventTime":"1970-01-01T00:00:00.000Z",
+      "eventName":"ObjectCreated:Put",
+      "userIdentity":{
+        "principalId":"AIDAJDPLRKLG7UEXAMPLE"
+      },
+      "requestParameters":{
+        "sourceIPAddress":"127.0.0.1"
+      },
+      "responseElements":{
+        "x-amz-request-id":"C3D13FE58DE4C810",
+        "x-amz-id-2":"FMyUVURIY8/IgAtTv8xRjskZQpcIZ9KG4V5Wp6S7S/JRWeUWerMUE5JgHvANOjpD"
+      },
+      "s3":{
+        "s3SchemaVersion":"1.0",
+        "configurationId":"testConfigRule",
+        "bucket":{
+          "name":"picture1gc",
+          "ownerIdentity":{
+            "principalId":"A3NL1KOZZKExample"
+          },
+          "arn":"arn:aws:s3:::picture1gc"
+        },
+        "object":{
+          "key":"lakeOfTheAngels.jpg",
+          "size":1024,
+          "eTag":"d41d8cd98f00b204e9800998ecf8427e",
+          "versionId":"096fKKXTRTtl3on89fVO.nfljtsv6qko"
+        }
+      }
+    }
+  ]
+}
```

Comment:
This looks really great Garrett, but I am going to need to see that README updated. All that documentation that is given in the instructions is what I am grading you on. This is what I am going to need to see added to your README:
- A description of how you used your lambda
- A description of any issues that you encountered during the deployment
- An image and thumbnail that your lambda processed
I would really like to see the README formatted out in proper Markdown as well please